src/progressive_automations_python/prefect_flows.py

```diff
@@ -0,0 +1,168 @@
+"""
+Prefect flow for desk lifter control.
+
+Provides workflow orchestration using Prefect for remote desk control.
+"""
+
+from prefect import flow, task
+from prefect.logging import get_run_logger
+
+from progressive_automations_python.desk_controller import (
+    move_to_height, 
+    check_duty_cycle_status_before_execution
+)
+from progressive_automations_python.duty_cycle import check_movement_against_duty_cycle, load_state
+from progressive_automations_python.config import UP_RATE, DOWN_RATE
+
+# Decorate core functions as tasks
+move_to_height_task = task(move_to_height)
+check_duty_cycle_status_task = task(check_duty_cycle_status_before_execution)
+
+
+@flow
+def simple_movement_flow(target_height: float):
```

Comment:
Even if the duty cycle is up and a rest is needed, you should still allow the command to go through. No errors raised. It will just take longer to execute. That's fine. Condense the logic here.